evennia/utils/evmenu.py

```diff
@@ -64,10 +64,41 @@ def another_node(caller, input_string, **kwargs):
 The return values must be given in the above order, but each can be
 returned as None as well. If the options are returned as None, the
 menu is immediately exited and the default "look" command is called.
-
-    text (str, tuple or None): Text shown at this node. If a tuple, the
-        second element in the tuple is a help text to display at this
-        node when the user enters the menu help command there.
+    text (str, tuple, dict or None): Text shown at this node. If a dict
+        it can also contain various formatting instructions. In such a
+        case the followng keys are already supported:
+            - 'text' (str): The text of the node.
+            - 'help' (str): The help text of the node.
+            - 'header' (str): Header text for the node.
+            - 'footer' (str): Footer text for the node.
+            - 'format' (str): String dictating the layout of the node if
+                the default layout is not being used. It currently
+                supports three options:
+                    - 'suppress': The node is not drawn at all
+                    - 'no_bars': The node is drawn similarly to the
+                        default format but without separator bars.
+                    - 'invert': The node is drawn similar to the
+                        default format but the options table will be
+                        above the text.
+            - 'optionformat' (dict): A dictionary containing information
```

Comment:
If using the `formatter` callable suggestions above, these would all basically become quick-options for the default formatters only (the **kwarg to send into default formatters). So maybe something like `formatoptions` would be a better name in that case. One could also imagine appending these values to all **kwargs sent to custom formatters too.

Reply:
I've changed 'optionsformat' to simply be part of the 'options' dict.

e.g.:
![Example_Options](https://user-images.githubusercontent.com/8743461/83319021-601d2580-a1ef-11ea-9ef4-bd0c0460ee17.png)
This keeps consistency with how all other parts (header, footer, text, etc.) of the node are defined which will hopefully increase ease of use.
(This design methodology means that if someone creates a formatter where they want to pass additional parameters they can easily pass those parameters by just expanding the corresponding dict)